docs/privacy-policy.md

```diff
@@ -0,0 +1,127 @@
+Privacy Policy
+==============
+
+**DRAFT**
+
+Your personal privacy is an important factor that the Sempervirens project and Numfocus 
+(our legal entity) considers in the development of each of our products and services. 
+We are committed to gathering NO personal information through sempervirens. We are 
+committed to being transparent and open. This Privacy Policy explains generically how 
+we receive information from you, and what we do with that  information once we have it.
+
+What do we mean by "personal information?"
+------------------------------------------
+
+The phrase "personal information" means information which identifies you, such as your 
+name, email address, twitter account names, or other user names. Any information that 
+falls outside of this is "non-personal information."
+
+Combining non-personal information with personal information we consider to be 
+"personal information."  Such combinations will not be collected. Removing all 
+personal information creates a data set that is deemed "non-personal information" 
+and may be collected.
+
+How do we learn information about you?
+---------------------------------------
+
+We learn non-personal information about you when:
+
+* you give it to us directly (e.g., when you choose to send us crash reports);
+* we collect it automatically through our products and services (e.g., the Sempervirens 
+  code); or
+* when you have opted-out of sending other non-personal information, we record that an 
+  opt-out event has occurred.
+
+
+What do we do with your information once we have it?
+----------------------------------------------------
+
+When you give us information, we will use it in the ways for which you've given us 
+permission. Generally, we use your information to help us provide and improve our 
+products and services to you and for you. This includes aggregations of the information 
+to display data such as global usage statistics.
+
+What are usage statistics?
+--------------------------
+
+Usage statistics or "telemetry" is a feature of Sempervirens that sends Numfocus usage, 
+performance, and responsiveness statistics about user interface features, application 
+program interfaces (APIs), application binary interfaces (ABIs), memory, hardware 
+configuration. Your IP address is also collected as a part of a standard web log. 
```

Comment:
On Thu, Jul 16, 2015 at 8:41 AM, Anthony Scopatz <notifications@github.com>
wrote:

> In docs/privacy-policy.md
> <https://github.com/njsmith/sempervirens/pull/16#discussion_r34802231>:
>
> > +
> > +What do we do with your information once we have it?
> > +----------------------------------------------------
> > +
> > +When you give us information, we will use it in the ways for which you've given us
> > +permission. Generally, we use your information to help us provide and improve our
> > +products and services to you and for you. This includes aggregations of the information
> > +to display data such as global usage statistics.
> > +
> > +What are usage statistics?
> > +--------------------------
> > +
> > +Usage statistics or "telemetry" is a feature of Sempervirens that sends Numfocus usage,
> > +performance, and responsiveness statistics about user interface features, application
> > +program interfaces (APIs), application binary interfaces (ABIs), memory, hardware
> > +configuration. Your IP address is also collected as a part of a standard web log.
>
> IP addresses are not personally identifiable on their own typically (see
> https://en.wikipedia.org/wiki/Personally_identifiable_information). Also,
> this could be made clearer that we are collecting the IPs as part of the
> normal weblog procedure. Maybe we should state more explicitly that we are
> not publishing individual IP addresses, but will store and publish
> aggregates of IPs. Does this sound right?
>
> From the wikipedia article you linked: "... for example, a user's IP
address as used in a communication exchange is classed as PII regardless of
whether it may or may not on its own be able to uniquely identify a person",
but I am not sure what constitues a "communication exchange". I'd think
sending data over the wire would count. See also:
http://googlepublicpolicy.blogspot.com/2008/02/are-ip-addresses-personal.html,
and comments therein. In particular, it seems that the EU thinks that IP
are definitely PII (and see some twists on that here:
http://moritzlaw.osu.edu/students/groups/is/files/2012/02/Lah_Formatted_Final.pdf;
warning: legalese...)

Also, not sure what "the normal weblog procedure is". Users would not be
literally visiting a website when the software they use makes calls to SV,
but their IP would still be recorded. We may or may not even record their
IP if they do not opt-in (I think that's what Mozilla does?).

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/njsmith/sempervirens/pull/16/files#r34802231>.
>